src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -264,11 +332,13 @@ private String generateName(@NotNull final Random rand)
         final String firstName;
         if (!female)
         {
-            firstName = getRandomElement(rand, Configurations.maleFirstNames);
+            firstName = String.format("%s %s. %s", getRandomElement(rand, Configurations.femaleFirstNames), getRandomLetter(rand),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "%s %s. %s" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)